packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.Props.ts

```diff
@@ -52,4 +52,9 @@ export interface ISearchBoxProps extends React.Props<SearchBox> {
    * @defaultvalue labelText
    */
   ariaLabel?: string;
+
+  /**
+   * The boolean which indicates whether to hide search icon.
```

Comment:
Hmm you are right. I didn't realize that there is a TextField control that also has onChange callback. Will try that.